src/openapi.json

```diff
@@ -506,18 +506,25 @@
 						"readOnly": true,
 						"example": 3011
 					},
-					"label": {
+					"defaultLabel": {
 						"type": "string",
 						"example": "First Name"
 					},
-					"description": {
+					"defaultDescription": {
 						"type": "string",
 						"example": "The first name of the applicant"
 					},
 					"shortCode": {
 						"type": "string",
 						"example": "firstName"
 					},
+					"localizations": {
+						"type": "object",
+						"additionalProperties": {
```

Comment:
I wanted to check if we could be stronger here, and somehow say that the property keys would be language tags -- this exists: https://json-schema.org/understanding-json-schema/reference/object#propertyNames

I don't think we can (particularly without listing the tags out in our openapi.json).  That said, I think object would warrant having a `description` explaining that the keys can be expected to be IETF language tags.